setup.py

```diff
@@ -10,7 +10,7 @@
       author='James D. Mitchell, Nicolas Thiéry',
       author_email='jdm3@st-andrews.ac.uk, Nicolas.Thiery@u-psud.fr',
       license='GPL3',
-      install_requires=['cppyy'],
+      install_requires=['cppyy','matplotlib','networkx'],
```

Comment:
I also don't see why matplotlib is required, please remove this too? Thanks!

Reply:
We needed matplotlib to show the graph but now that we are just return a networkx graph object, it's fine.